frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -90,50 +87,73 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
           <h5>Search for Parcel</h5>
         </Col>
         <Col md={6}>
-          <PidPinForm
-            handlePidChange={handlePidChange}
-            handlePinChange={handlePinChange}
-            nameSpace={nameSpace}
-          />
-          <CenteredSearchButton
-            onClick={(e: any) => {
-              e.preventDefault();
-              let newValues = {
-                ...formikProps.values,
-              };
-              newValues = setIn(newValues, nameSpace, getInitialValues());
-
-              formikProps.resetForm({
-                values: newValues,
-              });
-
-              if (pid?.length > 0) {
-                handlePidChange(pid, nameSpace);
-              } else if (pin?.length > 0) {
-                handlePinChange(pin, nameSpace);
-              } else if (geocoderResponse) {
-                handleGeocoderChanges(geocoderResponse, nameSpace);
-              }
-            }}
-          />
-          <Form.Row className="justify-content-end">
+          <Form.Row>
```

Comment:
the search fields are now separate from the actual form data used by the PIMS backend. That means that the search fields no longer need to be required fields, and that the actual data is displayed below the search fields.